### ISSUE TITLE
SBT <1.1.1 error: NoSuchMethodError appendWithSession

### ISSUE THREAD

**Issue by cbrunnkvist** [stryker-mutator/stryker4s#507]
### Summary

When I run `sbt stryker` in _some_ projects

it fails with an error `java.lang.NoSuchMethodError: sbt.Extracted.appendWithSession`

Stacktrace:

```
[info] Loading global plugins from /Users/cbrunnkvist/.sbt/1.0/plugins
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /Users/cbrunnkvist/some-repo/project/project
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /Users/cbrunnkvist/some-repo/project
[info] Loading settings from version.sbt,build.sbt ...
09:27:21.790 [pool-16-thread-1] INFO   - Set stryker4s logging level to info
09:27:21.929 [pool-16-thread-1] INFO  stryker4s.config.ConfigReader$ - Attempting to read config from /Users/cbrunnkvist/some-repo/stryker4s.conf
[error] java.lang.NoSuchMethodError: sbt.Extracted.appendWithSession(Lscala/collection/Seq;Lsbt/State;)Lsbt/State;
[error] 	at stryker4s.sbt.runner.SbtMutantRunner.<init>(SbtMutantRunner.scala:76)
[error] 	at stryker4s.sbt.Stryker4sSbtRunner.resolveRunner(Stryker4sSbtRunner.scala:19)
[error] 	at stryker4s.run.Stryker4sRunner.run(Stryker4sRunner.scala:21)
[error] 	at stryker4s.run.Stryker4sRunner.run$(Stryker4sRunner.scala:14)
[error] 	at stryker4s.sbt.Stryker4sSbtRunner.run(Stryker4sSbtRunner.scala:17)
[error] 	at stryker4s.sbt.Stryker4sMain$.$anonfun$strykerTask$1(Stryker4sMain.scala:36)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:39)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:66)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:262)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:271)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:262)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:174)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:36)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (stryker) java.lang.NoSuchMethodError: sbt.Extracted.appendWithSession(Lscala/collection/Seq;Lsbt/State;)Lsbt/State;
```

I suspect that it is some problem with this multi-project SBT setup, or a version incompatibility, but I would recommend we check for this and log a more informative error.

### Stryker4s config

```hocon
stryker4s {
  base-dir: "fd-core"
  mutate: [ "src/main/scala/com/agoda/terracotta/core/model/score/*.scala" ]
}
```

### Stryker4s environment

```
stryker4s 0.8.1
```

### Your Environment
| software             | version(s) |
| -------------------- | ---------- |
| Scala version        | `scalaVersion := "2.11.8"` |
| Build tool & version | `sbt.version = 1.1.0` |
| Operating System     | `macOS 10.15.4` |

**Comment by cbrunnkvist** [comment]
Judging from the comments on [SBT #4048](https://github.com/sbt/sbt/issues/4048#issuecomment-376935512), it has to do with Stryker4s using a method that was not added until 1.1.1 and these problematic projects all use 1.1.0

I think it would be a great improvement for early adopters if the SBT plugin could check for version requirements before it starts.

**Comment by hugo-vrijswijk** [comment]
Thanks for opening this issue and sorry for the late reply 🙈. I agree a version check is probably a good thing to have! Pinning down exactly what version stryker4s supports could be a little tricky. But once we have it we could add a sbt-scripted test to ensure compatibility in the future

**Comment by hugo-vrijswijk** [comment]
@cbrunnkvist This feature is now available in the latest [`v0.9.1` release](https://github.com/stryker-mutator/stryker4s/releases/tag/v0.9.1)
